notebooks/cookbook/Querying and Modifying Charges and Other Parameters.ipynb

```diff
@@ -98,12 +98,81 @@
     "    if particle1 == 0 or particle2 == 0:\n",
     "        print(f'Particles ({particle1}, {particle2}), length = {length}, k = {k}')"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "50f5157c",
+   "metadata": {},
+   "source": [
+    "In some cases you may want to modify those parameters to make a bond behave differently from what the `ForceField` assigned.\n",
+    "We show you here that to modify these parameters are also easy."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cd5396c5",
+   "metadata": {},
+   "source": [
+    "Let's say we would want to modify the force constant of the bond connecting particle 1 and 0 to a new number `_k`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "0dc0db66",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "k_ = 2666 * unit.kilojoule / ( unit.nanometer ** 2 * unit.mole )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "086c7629",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i in range(bonded.getNumBonds()):\n",
+    "    particle1, particle2, length, k = bonded.getBondParameters(i)\n",
+    "    if particle1 == 1 and particle2 == 0:\n",
+    "        bonded.setBondParameters(i, particle1, particle2, length, k_)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f9728ad",
+   "metadata": {},
+   "source": [
+    "Now if you query again you can see new parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "68498ff5",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Particles (1, 0), length = 0.101 nm, k = 2666.0 kJ/(nm**2 mol)\n"
+     ]
+    }
+   ],
+   "source": [
+    "bonded = [f for f in system.getForces() if isinstance(f, HarmonicBondForce)][0]\n",
+    "for i in range(bonded.getNumBonds()):\n",
+    "    particle1, particle2, length, k = bonded.getBondParameters(i)\n",
+    "    if particle1 == 1 and particle2 == 0:\n",
```

Comment:
Let's make the condition the same as before: `particle1 == 0 or particle2 == 0`.  That way it will print out the same list of bonds, and they can see that only one has changed.